### PR TITLE
Made validation function

### PR DESCRIPTION
**Pull request by lochem2-718**
also added printing to stderr and appropriate return value on error

### CODE REVIEW

**Comment by igorsteinmacher** [comment]
Well done
Group 8 will review this

**Review by DanielRustrum — COMMENTED** [review]
I suggest that you either write more self documenting code or adding documentation within the code. It is hard to parse and walk through

**Review by DanielRustrum — COMMENTED** [review]
Resolve conflicts 😁

File: assignmnetPR/assignmnetPR.c [hunk 1]
```diff
@@ -36,26 +55,30 @@ bool mantissa(char numString[], int *numerator, int *denominator)
     int index = 0;
```
Comment (andresgsepulveda) [inline assignmnetPR/assignmnetPR.c:55]: Some documentation on what the function is going to do would be helpful. A basic "purpose", "parameters" and "output" type situation would help us to understand from a review standpoint the logic associated.

File: assignmnetPR/assignmnetPR.c [hunk 2]
```diff
@@ -1,31 +1,50 @@
...
+    while (*curr_char != '\0' && (is_digit(*curr_char) || *curr_char == '.' || *curr_char == '-'))
+    {
+        curr_char++;
+    }
+    return *curr_char == '\0';
+}
+
 bool characteristic(char numString[], int *c)
 {
     int index = 0;
```
Comment (biancatrink) [inline assignmnetPR/assignmnetPR.c:22]: - Variable names:
I suggest using names for variables that explain what they are used for. 
For example, cval and c could be changes to more specific ones.
Another suggestion regarding variable names is to follow a pattern for variable names.
For example, the variable signValue follows a pattern of having two words (sign and Value) and the first letter of the second word is uppercase (Value).

**Comment by biancatrink** [comment]
- Variable names:
I suggest using names for variables that explain what they are used for. 
For example, cval and c could be changes to more specific ones.
Another suggestion regarding variable names is to follow a pattern for variable names.
For example, the variable signValue follows a pattern of having two words (sign and Value) and the first letter of the second word is uppercase (Value).

**Review by elijahab — APPROVED** [review]

**Review by elijahab — APPROVED** [review]
